src/components/WorkerListItem/index.tsx

```diff
@@ -2,43 +2,72 @@
 
 import React from 'react';
 
+import { toast } from 'react-toastify';
+
 import * as S from './style';
 
-import { EmailIcon, SNSIcon } from '@/assets';
+import {
+  EmailIcon,
+  NotExistEmailIcon,
+  NotExistSNSIcon,
+  SNSIcon,
+} from '@/assets';
```

Comment:
```suggestion
import * as I from '@/assets';
```

긴 import문을 간결하게 변경할 수 있어 보입니다!

Reply:
늘어난 import 문을 확인 못 했네요 좋은 제안 감사합니다!

Reply:
[6b79393](https://github.com/GSM-Networking/GSM-Networking-client/pull/7/commits/6b793930cd5c904112cb0a1a59f672d726242001)

수정했습니다